hello-world-python/settings.gradle

```diff
@@ -0,0 +1,27 @@
+pluginManagement {
+    repositories {
+        maven {
+            def repoPath = file(rootDir.relativePath(file(conclaveRepo)))
+            if (repoPath == null) {
+                throw new Exception("Make sure the 'conclaveRepo' setting exists in gradle.properties, or your " +
+                        "$HOME/gradle.properties file. See https://docs.conclave.net/gradle-properties.html")
```

Comment:
It should be \$HOME instead of $HOME. The variable HOME does not exist and without the backwards slash it will cause an error.

Reply:
done.